architecture/2024-08-applicationdeployment-controller.md

```diff
@@ -0,0 +1,370 @@
+# ApplicationDeployment Controller
```

Comment:
One high level question have I ....

Is `ApplicationDeployment` the right thing? `ApplicationDeployment` implies that it's for using Bicep to deploy Radius applications. 

What about other usecases for Bicep that aren't an application?

Reply:
(I know *I was the one* that created `ApplicationDeployment` in my prototype. I'm not 100% sure that's the right terminology or design.)